src/arithmetic/arithmetic_expression.c

```diff
@@ -498,7 +503,7 @@ AR_ExpNode *AR_EXP_Clone(AR_ExpNode *exp) {
 		assert(false);
 		break;
 	}
-
+	clone->resolved_name = exp->resolved_name;
```

Comment:
No need to duplicate?

Reply:
No; although `resolved_name` looks like the result of `AR_EXP_ToString`, it is a `const` parser artifact that gets freed with the `parse_result` after execution.